server/graphql/v2/object/Host.js

```diff
@@ -437,6 +464,135 @@ export const Host = new GraphQLObjectType({
           };
         },
       },
+      contributionStats: {
+        type: new GraphQLNonNull(ContributionStats),
+        args: {
+          account: {
+            type: new GraphQLList(new GraphQLNonNull(AccountReferenceInput)),
+            description: 'A collection of accounts for which the contribution stats should be returned.',
+          },
+          dateFrom: {
+            type: GraphQLDateTime,
+            description: 'Calculate contribution statistics beginning from this date.',
+          },
+          dateTo: {
+            type: GraphQLDateTime,
+            description: 'Calculate contribution statistics until this date.',
+          },
+        },
+        async resolve(host, args, req) {
+          if (!req.remoteUser?.isAdmin(host.id)) {
+            throw new Unauthorized('You need to be logged in as an admin of the host to see the contribution stats.');
+          }
+          const where = {
+            HostCollectiveId: host.id,
+            kind: TransactionKind.CONTRIBUTION,
+            type: TransactionTypes.CREDIT,
+            isRefund: false,
+            RefundTransactionId: null,
+          };
+          const numberOfDays = getNumberOfDays(args.dateFrom, args.dateTo, host);
+          const dateRange = getFilterDateRange(args.dateFrom, args.dateTo);
+          if (dateRange) {
+            where.createdAt = dateRange;
+          }
+          if (args.account) {
+            const collectives = await fetchAccountsWithReferences(args.account, { throwIfMissing: true });
```

Comment:
Small optimization if you rebase on `main`

```suggestion
            const collectives = await fetchAccountsWithReferences(args.account, { throwIfMissing: true, attributes: ['id'] });
```